pkg/datapath/linux/linux_defaults/mark.go

```diff
@@ -46,6 +50,18 @@ const (
 	// to a proxy.
 	MagicMarkIsToProxy uint32 = 0x0200
 
+	// MagicMarkIngressEPID determines that the traffic is sourced from the
+	// proxy which is capturing traffic before it is subject to ingress
+	// policy enforcement that must be done after the proxy. The identity
+	// stored in the mark is destination Endpoint ID.
+	MagicMarkIngressEPID int = 0x0800
```

Comment:
Does this not overlap with `MagicMarkK8sDrop`? Does that matter?

Reply:
No, this is in a different nibble (`0x8000`). All four bits covered by `0x0F00` are used by Cilium.